lib/events.js

```diff
@@ -354,6 +376,16 @@ class AdobeIOEvents {
             retryOptions: retryOptions
         });
 
+        if (response.status === 409) {
+            console.error("Journal already exists, returning existing");
+            const conflictingRegistrationId = response.headers['x-conflicting-id'];
+            if (!conflictingRegistrationId) {
+                console.error("Could not find conflicting registration id in response headers");
+                throw new Error("Could not find conflicting registration id in response headers");
+            }
+            return this.getJournal(journal, conflictingRegistrationId);
```

Comment:
it looks cleaner to do `return handleFetchErrors(response).json();` at one place instead of inside `getJournal()`. Also please structure new Error to be consistent.

Reply:
My understanding was that it would give clients the option to directly invoke the `getJournal` API.